pytorch_lightning/utilities/argparse.py

```diff
@@ -285,8 +285,10 @@ def _parse_args_from_docstring(docstring: str) -> Dict[str, str]:
     return parsed
 
 
-def _gpus_allowed_type(x: str) -> Union[int, str]:
-    if "," in x:
+def _gpus_allowed_type(x: str) -> Union[int, str, None]:
+    if x.lower() == "none":
```

Comment:
with this change we should also update the table in the section [here](https://pytorch-lightning.readthedocs.io/en/latest/advanced/multi_gpu.html#select-gpu-devices).